src/main/java/io/cdap/plugin/gcp/gcs/connector/GCSConnector.java

```diff
@@ -97,7 +97,8 @@ public void test(ConnectorContext context) throws ValidationException {
     Credentials credentials = null;
     try {
       credentials =
-        GCPUtils.loadServiceAccountCredentials(config.getServiceAccount(), config.isServiceAccountFilePath());
+        config.getServiceAccount() == null ? null :
+          GCPUtils.loadServiceAccountCredentials(config.getServiceAccount(), config.isServiceAccountFilePath());
```

Comment:
what will happen when you get the GCS client with null credentials?

Reply:
it won't set the credentials if it is null, https://github.com/data-integrations/google-cloud/blob/develop/src/main/java/io/cdap/plugin/gcp/common/GCPUtils.java#L173

Reply:
sg :)